### PR TITLE
Create light-install.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Added Light install guide to Getting Started.

### CODE REVIEW

File: source/help/getting-started/light-install.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+Install Mattermost using `Mattermost Omnibus <https://docs.mattermost.com/install/mattermost-omnibus.html>`_.
+
+Open a terminal window and enter: 
+
+.. code-block::
+
+  curl -o- deb.packages.mattermost.com/repo-setup.sh | sudo bash
+
+Note: Although the recommended way to install and configure Omnibus is with SSL enabled, if you want to use or test without it, you can run:
```
Comment (jasonblais) [inline source/help/getting-started/light-install.rst:23]: Do we want to render this as a note with the `.. note::` RST formatting?

File: source/help/getting-started/light-install.rst [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+  curl -o- deb.packages.mattermost.com/repo-setup.sh | sudo bash
+
+Note: Although the recommended way to install and configure Omnibus is with SSL enabled, if you want to use or test without it, you can run: 
+
+.. code-block:: 
+  
+  sudo MMO_HTTPS=false apt install mattermost-omnibus.
+
+When your server is up and running, navigate to it via the domain name that points to your server (or the server’s IP address if you’re not using a domain name). Next, create a team and invite people to join your Mattermost server.
```
Comment (jasonblais) [inline source/help/getting-started/light-install.rst:29]: is there an example we can give for the domain name, or would that be confusing?

**Review by jasonblais — APPROVED** [review]
Just two non-blocking comments, great work Justine!

**Comment by jasonblais** [comment]
Also, once merged, I'll add a marketing ticket to add a link to it from the download page
image

File: source/help/getting-started/light-install.rst (hunk 1 above)
Comment (justinegeffen) [reply source/help/getting-started/light-install.rst]: Good catch, thanks! It's been updated.

File: source/help/getting-started/light-install.rst [hunk 3]
```diff
@@ -0,0 +1,57 @@
+Light Install Guide
+===================
+
+Mattermost is a messaging and collaboration platform. With Mattermost, you can integrate the tools you use every day into one place and never miss a notification or task. 
+
+Before you get started you'll need:
+
+* A clean Ubuntu server (18.04 or 20.04) with root level access
+* (Recommended) A domain name pointing to your server
```
Comment (justinegeffen) [inline source/help/getting-started/light-install.rst:9]:
```suggestion
* (Recommended) A domain name pointing to your server (e.g. ``mymattermostserver.com``)
```

File: source/help/getting-started/light-install.rst [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+.. note::
+  
+  Although the recommended way to install and configure Omnibus is with SSL enabled, if you want to use or test without it, you can run: 
+
+.. code-block:: 
+  
+  sudo MMO_HTTPS=false apt install mattermost-omnibus.
+
+When your server is up and running, navigate to it via the domain name that points to your server (or the server’s IP address if you’re not using a domain name). Next, create a team and invite people to join your Mattermost server.
```
Comment (justinegeffen) [inline source/help/getting-started/light-install.rst:31]:
```suggestion
When your server is up and running, navigate to it via the domain name (e.g. ``mymattermostserver.com``) that points to your server (or the server’s IP address if you’re not using a domain name). Next, create a team and invite people to join your Mattermost server.
```

**Comment by justinegeffen** [comment]
@mgdelacroix, can you confirm that the domain name example/instructions are correct? Thank you!

File: source/help/getting-started/light-install.rst [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+Open a terminal window and enter: 
+
+.. code-block::
+
+  curl -o- deb.packages.mattermost.com/repo-setup.sh | sudo bash
+
+.. note::
+  
+  Although the recommended way to install and configure Omnibus is with SSL enabled, if you want to use or test without it, you can run:
```
Comment (mgdelacroix) [inline source/help/getting-started/light-install.rst:25]: We should indicate the recommended way to install before the testing one. Maybe adding a paragraph on top of this note like:

```rst
Then, to install the Omnibus package, run:

.. code-block:

  sudo apt install mattermost-omnibus
```

File: source/help/getting-started/light-install.rst [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+  curl -o- deb.packages.mattermost.com/repo-setup.sh | sudo bash
+
+.. note::
+  
+  Although the recommended way to install and configure Omnibus is with SSL enabled, if you want to use or test without it, you can run: 
+
+.. code-block:: 
+  
+  sudo MMO_HTTPS=false apt install mattermost-omnibus.
```
Comment (mgdelacroix) [inline source/help/getting-started/light-install.rst:29]:
```suggestion
  sudo MMO_HTTPS=false apt install mattermost-omnibus
```

**Review by mgdelacroix — COMMENTED** [review]
Added a couple of small comments. Otherwise LGTM! :rocket:

File: source/help/getting-started/light-install.rst (hunk 5 above)
Comment (justinegeffen) [reply source/help/getting-started/light-install.rst:25]: Ah, good catch, thank you!

**Review by amyblais — COMMENTED** [review]
I assume this won't need to be added to TOC. Otherwise looks ok.

**Comment by justinegeffen** [comment]
> I assume this won't need to be added to TOC. Otherwise looks ok.

My bad, it should be added to TOC. Will add that now.

**Comment by justinegeffen** [comment]
@jasonblais, I've added a new section in the TOC because this guide didn't fit in with the current content. 0/5 on whether it's the best option and happy to discuss alternatives.

**Comment by jasonblais** [comment]
@justinegeffen Another option is the "Getting Started" section in the admin guide?

**Review by jasonblais — APPROVED** [review]

**Comment by jasonblais** [comment]
@justinegeffen is this ready for final reviews?

**Review by amyblais — APPROVED** [review]

**Review by jasonblais — APPROVED** [review]

**Comment by jasonblais** [comment]
@mgdelacroix as well for final reviews :)

**Review by mgdelacroix — APPROVED** [review]
LGTM! :+1:

**Pull request merged**